packages/cli/src/active-executions.ts

```diff
@@ -215,6 +215,8 @@ export class ActiveExecutions {
 			executionIds.forEach((executionId) => this.stopExecution(executionId));
 		}
 
+		if (config.getEnv('executions.mode') === 'queue') return;
```

Comment:
since we won't wait for executions to finish, any execution that has response promise will now return a 5xx. Is that intentional?

Reply:
Good point, removed. In future, we can try to tighten this by only waiting for executions with response promises.